Providers/ItkPrometheusServiceProvider.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+namespace Modules\ItkPrometheus\Providers;
+
+use Illuminate\Support\Arr;
+use Illuminate\Support\ServiceProvider;
+use Illuminate\Database\Eloquent\Factory;
+use Modules\ItkPrometheus\Service\PrometheusService;
+use Prometheus\CollectorRegistry;
+use Prometheus\Storage\Redis;
+
+define('ITK_PROMETHEUS_MODULE', 'itkprometheus');
+
+require_once __DIR__.'/../vendor/autoload.php';
+
+class ItkPrometheusServiceProvider extends ServiceProvider
+{
+    /**
+     * Indicates if loading of the provider is deferred.
+     *
+     * @var bool
+     */
+    protected $defer = false;
+
+    /**
+     * Boot the application events.
+     *
+     * @return void
+     */
+    public function boot()
+    {
+        $this->registerConfig();
+        $this->registerViews();
+        $this->registerFactories();
+        $this->loadMigrationsFrom(__DIR__ . '/../Database/Migrations');
+        $this->hooks();
```

Comment:
Why have this call to an empty `hooks()` function... if real need please comment why